contrib/automation_tests/orbit_tracks.py

```diff
@@ -16,10 +16,11 @@ def main(argv):
         ConnectToStadiaInstance(),
         FilterAndSelectFirstProcess(process_filter="hello_ggp"),
         Capture(),
-        # "sdma0" is not present on the DevKits, instead there is "vce0", so this tests for "sdma0 or vce0"
+        # We check for either "sdma0" or "vce0", to exist. Both are connected to the encoding of video frames.
         VerifyTracksExist(track_names=[
```

Comment:
Should this be split into two calls of VerifyTracksExist, one allowing duplicates and only checking for *sdma0* or *vce0*, the other checking for the others and disallowing duplicates?